### DISCUSSION TITLE
WS2812 control via MQTT from  Node-red

### DISCUSSION THREAD

**Discussion by myozone** [arendst/Tasmota#22322] [General]
I have a ESP8266 that has a name of server, I want to send it commands for WS2812 LEDs. It responds to send commands to topic home/tasmota/ledcontrol but not cmnd/server ?

The command is a simple Node-red inject node with
```
msg.topic = home/tasmota/ledcontrol (works but cmnd/server doesn't)
msg.payload  {     "color": "#00FF00",     "brightness": 80,     "effect": "blink" }

Topic = home/tasmota/ledcontrol  responds with:
stat/server/RESULT = {"POWER5":"ON","Dimmer":100,"Color":"00FF00","HSBColor":"120,100,100","Channel":[0,100,0]}

but Topic = cmnd/server or cmnd/server/ledcontrol responds with:
stat/server/RESULT = {"Command":"Unknown"}
```

Any ideas ?

**Comment by sfromis** [comment]
Well, it helps to use [commands](https://tasmota.github.io/docs/Commands/#light) supported by Tasmota, which does **not** include something named `ledcontrol`. IDK what you have subscribing to `home/tasmota/#`, but it does not look like it is Tasmota. The standard Tasmota setup includes topic prefixes of `cmnd`, `stat` and `tele`. Not `home`. Maybe you have some Node-Red setup doing something extra before passing to Tasmota on a different topic, and different message structure. That payload with `color`, `brightness` and `effect` is also not even similar to how Tasmota works.

**Reply by myozone** [reply]
You're absolutely correct, I had a another flow with home/tasmota/ledcontrol as it's MQTT input that I forgot about that was messing things up ! -  It had me really foxed for a bit. 

Thanks for making me think.

I have 50+ flows with many interactions
